eos/b-decays/b-to-dstar-l-nu.cc

```diff
@@ -19,18 +19,183 @@
+            inline double v4()      const   { return _vv[3]; }
+            inline double v5()      const   { return _vv[4]; }
+            inline double v6()      const   { return _vv[5]; }
+            inline double v7()      const   { return _vv[6]; }
+            inline double v8()      const   { return _vv[7]; }
+            inline double v9()      const   { return _vv[8]; }
+            inline double v10()     const   { return _vv[9]; }
+            inline double v11()     const   { return _vv[10];}
+            inline double v12()     const   { return _vv[11];}
+
+            // longitudinal polarization amplitude
+            inline double normalized_amplitude_polarization_L() const
+            {
+                return  vv10() - vv20() / 3.0;
+            }
+
+            // transverse polarization amplitude
+            inline double normalized_amplitude_polarization_T() const
+            {
+                return  2.0 * (vv1T() - vv2T() / 3.0);
+            }
+
+            // redefined decay width
+            inline double normalized_decay_width() const
+            {
+                return 3.0 / 4.0 * ( normalized_amplitude_polarization_L() + normalized_amplitude_polarization_T() );
+            }
+
+            // polarization fraction
+            inline double f_L() const
+            {
+                return  normalized_amplitude_polarization_L() / (normalized_amplitude_polarization_L() + normalized_amplitude_polarization_T());
+            }
+
+            // a_fb leptonic
+            inline double a_fb_leptonic() const
+            {
+                return  (vv3T() + vv30() / 2.0) / (normalized_amplitude_polarization_L() + normalized_amplitude_polarization_T());
+            }
+
+            // transverse azimuthal asymmetries
+            inline double a_c_1() const
+            {
+                return  4.0 * vv4T() / (3.0 * (normalized_amplitude_polarization_L() + normalized_amplitude_polarization_T()));
+            }
+
+            inline double a_c_2() const
+            {
+                return  vv20T() / (normalized_amplitude_polarization_L() + normalized_amplitude_polarization_T());
+            }
+
+            inline double a_c_3() const
+            {
+                return  vv10T() / (normalized_amplitude_polarization_L() + normalized_amplitude_polarization_T());
+            }
+
+            // T-odd CP asymmetries
+            inline double a_t_1() const
+            {
+                return  4.0 * vv5T() / (3.0 * (normalized_amplitude_polarization_L() + normalized_amplitude_polarization_T()));
+            }
+
+            inline double a_t_2() const
+            {
+                return  vv30T() / (normalized_amplitude_polarization_L() + normalized_amplitude_polarization_T());
+            }
+
+            inline double a_t_3() const
+            {
+                return  vv40T() / (normalized_amplitude_polarization_L() + normalized_amplitude_polarization_T());
+            }
+        };
```

Comment:
Observables ok, but please add the 4-fold angular distribution in eq.(6) of DSD2014

Reply:
I think we can merge w/o the 4-fold distribution and add it later on!?

Reply:
@cBobeth and @dvandyk  I have now included also the 4-fold angular distribution, cheers